public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -117,9 +119,19 @@ export class LogRowMessage extends PureComponent<Props> {
               fill="text"
               size="md"
               getText={this.getLogText}
-              tooltip="Copy"
+              tooltip="Copy logline to clipboard"
               tooltipPlacement="top"
             />
+            {onPermalinkClick && row.uid && (
+              <IconButton
+                tooltip="Copy shortlink to logline"
```

Comment:
```suggestion
                tooltip="Copy shortlink to log line"
```

Reply:
Fixed in ad6baafe6937c8abeccacf29d3c2c19a37baa94f